src/lib.rs

```diff
@@ -70,6 +75,14 @@ macro_rules! broker_handler {
                 info!("{} {}", req.method, req.url);
                 Self::handle_body(&self.app, req)
                     .or_else(|e| handle_error(&self.app, req, e))
+                    .map(|mut res| {
+                        set_default_headers(&mut res);
+                        res
+                    })
+                    .map_err(|mut err| {
+                        set_default_headers(&mut err.response);
+                        err
+                    })
```

Comment:
Does Iron have a notion of middleware that we could register, instead of manually doing this on each handler?